docs/docs/lists-conditional-rendering.md

```diff
@@ -92,25 +115,57 @@ ReactDOM.render(<NumberList numbers={numbers} />, document.getElementById('root'
 
 **Example: Correct key usage:** _Assigns `key` inside `map()` function._
 
-```javascript{7}
+```javascript{9}
 function Number(props) {
   return <li>{props.value}</li>;
 }
 
 function NumberList(props) {
   const numbers = props.numbers
-  return <ul>{numbers.map((item) => <Number key={"item-" + props.value} value={item} />)}</ul>
+  return (
+    <ul>
+    {numbers.map((item) => <Number key={"item-" + item} value={item} />)}
+    </ul>
+  )
 }
 
 const numbers = [1, 2, 3, 4, 5];
-ReactDOM.render(<NumberList numbers={numbers} />, document.getElementById('root'));
+ReactDOM.render(
+  <NumberList numbers={numbers} />,
+  document.getElementById('root')
+);
 ```
 
-[Try it out on Codepen.](https://codepen.io/ericnakagawa/pen/Egpdrz?editors=0010)
+[Try it out on Codepen.](https://codepen.io/ericnakagawa/pen/zKmJWK?editors=0010)
 
 ## Conditional Rendering
 
-You can create distinct components that encapsulate behavior you need. Then, you can render only the specific component you need, depending on the state of your application.
+In React, you can create distinct components that encapsulate behavior you need. Then, you can render only the specific component you need, depending on the state of your application.
+
+In this example, we pass the variable `loggedIn` representing our state, together with `&&` operator, and the component we wish to render. Depending on the value of `loggedIn` the corresponding component will be rendered.
+
+```javascript
+function UserGreeting(props) {
+  return <div>Welcome back!</div>
+}
+function GuestGreeting(props) {
+  return <div>Please sign up.</div>
+}
+function VisitorMessage(props) {
+  var loggedIn = true; // change this to false
+  return (
+    <div>
+      {loggedIn && <UserGreeting />}
+      {!loggedIn && <GuestGreeting />}
+    </div>
+  );
+}
+
+ReactDOM.render(<VisitorMessage />, document.getElementById('root'));
+```
+
+[Try it out on Codepen](https://codepen.io/ericnakagawa/pen/WGaKLy?editors=0011#0)
+
```

Comment:
Note that `&&` is regular JS operator, and this works because React ignores false/null in render output.